docs/decisions/0012-initial-file-format.md

```diff
@@ -0,0 +1,113 @@
+    <td>n2</td>
+    <td>n3</td>
+  </tr>
+  <tr>
+    <td>20</td>
+    <td>35</td>
+    <td>62</td>
+  </tr>
+</table>
+
+The equivalent row format would be 
+
+<table>
+  <tr>
+    <td>1</td>
+    <td>n1</td>
+    <td>20</td>
+  </tr>
+  <tr>
+    <td>2</td>
+    <td>n2</td>
+    <td>35</td>
+  </tr>
+  <tr>
+    <td>3</td>
+    <td>n3</td>
+    <td>62</td>
+  </tr>
+</table>
+
+
+
+Pros: 
+- Generally speaking, good for storing large amounts data
+- Saves on cloud storage space by using highly efficient column-wise compression, and flexible encoding schemes for columns with different data types.
+- Well-supported file format from analytics tools
+
+Cons:
+- Readability without transformation is not as straightforward compared to a row-based format like CSV
+- Changing a schema over time is not as simple compared to CSV.
+
+
+### CSV
+CSV is one of the most common file format for reading data. It has a simple row-based structure that allows users to integrate easily with table readers such as Excel. 
+
+Pros: 
+- Supported by nearly all table readers a user may want to use.
+- Easily readable without transformation. 
+- Most developers understand how CSV files work
+
+Cons:
+- Costly to store. Because one row has multiple data types, it is harder to compress the files for storage
+- Slower to read. Because of disk space needed to read the file, it is slower to read through the CSV file.  
+
+### Fully-Managed DB
+A fully managed DB would circumvent the need to export data into a file format before querying the data. With RDBMS support, analysts could query data using SQL from our DB without having to manage files on their own. The export process would create an export function to create tables and write to tables in postgresql
+
+Pros: 
+- Direct querying our data without extra steps for analytics
+- More control for consumers to get the exact data that they need. 
+
+Cons:
+- High infrastructure costs by needing develops to stand up the resources and maintain it
+- Likely a more complex initial setup for security for STLTs to access the data they want. 
+
+## Decision Outcome
+
+**Parquet** 
+Because STLTs and the CDC may be storing large amounts of data. Using a file format that has advantages in storage costs will help minimize the cost of storage. In a chart from towardsdatascience.com, a CSV file of 1TB would be about a 0.25 TB file in parquet. These cost savings and performance gains outweigh the raw readability of CSV for our project. 
+
+In future iterations, we will support exports to CSV in addition parquet. 
+
+
+## Appendix (OPTIONAL)
+
+[CSV vs Parquet](https://towardsdatascience.com/csv-files-for-storage-no-thanks-theres-a-better-option-72c78a414d1d)
+[Pros and cons of parquet](https://stackoverflow.com/questions/36822224/what-are-the-pros-and-cons-of-parquet-format-compared-to-other-formats)
+[Internal Document on Parquet](https://docs.google.com/spreadsheets/d/1IPAElrM3RoPPFDHtjU3JSImBCXfMIO_NvwgzmKYkUHM/edit#gid=0)
```

Comment:
I think this appendix is great. However, I'm a little concerned about including the final doc because I don't believe we are making documents on this drive available to the public while this repo is most definitely public facing. Could we remove this reference, or move the document to another location that would be more appropriate to point to?